x-pack/plugin/security/qa/smoke-test-all-realms/src/javaRestTest/java/org/elasticsearch/xpack/security/authc/SecurityRealmSmokeTestCase.java

```diff
@@ -53,6 +53,8 @@ public abstract class SecurityRealmSmokeTestCase extends ESRestTestCase {
         .configFile("saml-metadata.xml", Resource.fromClasspath("saml-metadata.xml"))
         .configFile("kerberos.keytab", Resource.fromClasspath("kerberos.keytab"))
         .configFile("oidc-jwkset.json", Resource.fromClasspath("oidc-jwkset.json"))
+        .configFile("ldap_role_mapping.yml", Resource.fromClasspath("ldap_role_mapping.yml"))
```

Comment:
can you add a comment to the this configuration to mention that these are are not actually used and maybe rename to dummy_ldap_role_mapping.yml and the like ?